src/View/Cell.php

```diff
@@ -232,12 +232,13 @@ public function render($template = null)
      * @param string $action The action invoked.
      * @return array The cache configuration.
      */
-    protected function _cacheConfig($action)
+    protected function _cacheConfig($action, $template = null)
     {
         if (empty($this->_cache)) {
             return [];
         }
-        $key = 'cell_' . Inflector::underscore(get_class($this)) . '_' . $action;
+        $template = $template ?: "default";
+        $key = 'cell_' . Inflector::underscore(get_class($this)) . '_' . $action . '_' . $template;
```

Comment:
Using action name as well as template to generate the cache key seems unnecessary. How about just using template name? That would also avoid having to add new argument to the method.